src/js/components/location/LocationChooser.jsx

```diff
@@ -113,13 +112,24 @@ class LocationChooser extends Component {
 const mapStateToProps = state => ({
   currentLocationName: state.session.currentLocation.name,
   defaultTranslationsFetched: state.session.fetchedTranslations.default,
+  modal: state.session.modal,
```

Comment:
Should be a more fitting name for a boolean.

Reply:
Done